Tests.Unit.Core.ApplicationServices/Model/ItSystemUsageTest.cs

```diff
@@ -460,6 +487,119 @@ public void Valid_When_All_Null()
             Assert.True(validity.Result);
         }
 
+        [Fact]
+        public void AddPersonalData_Adds_PersonalData()
+        {
+            var itSystemUsage = new ItSystemUsage
+            {
+                SensitiveDataLevels = new List<ItSystemUsageSensitiveDataLevel>
+                {
+                    new() {SensitivityDataLevel = SensitiveDataLevel.PERSONALDATA}
+                }
+            };
+            var personalDataOption = A<GDPRPersonalDataOption>();
+
+            var error = itSystemUsage.AddPersonalData(personalDataOption);
+
+            Assert.False(error.HasValue);
+            Assert.Contains(personalDataOption, itSystemUsage.PersonalDataOptions.Select(x => x.PersonalData));
+        }
+
+        [Fact]
+        public void AddPersonalData_Returns_BadState()
+        {
+            var itSystemUsage = new ItSystemUsage();
+            var personalDataOption = A<GDPRPersonalDataOption>();
+
+            var error = itSystemUsage.AddPersonalData(personalDataOption);
+
+            Assert.True(error.HasValue);
+            Assert.Equal(OperationFailure.BadState, error.Value.FailureType);
+        }
+
+        [Fact]
+        public void AddPersonalData_Returns_NoError_If_Data_Already_Present()
+        {
+            var personalDataOption = A<GDPRPersonalDataOption>();
+            var itSystemUsage = new ItSystemUsage
+            {
+                SensitiveDataLevels = new List<ItSystemUsageSensitiveDataLevel>
+                {
+                    new() {SensitivityDataLevel = SensitiveDataLevel.PERSONALDATA}
+                },
+                PersonalDataOptions = new List<ItSystemUsagePersonalData>()
+                {
+                    new(){PersonalData = personalDataOption}
+                }
+            };
+
+            var error = itSystemUsage.AddPersonalData(personalDataOption);
+
+            Assert.False(error.HasValue);
+            Assert.Contains(personalDataOption, itSystemUsage.PersonalDataOptions.Select(x => x.PersonalData));
+        }
+
+        [Fact]
+        public void RemovePersonalData_Removes_PersonalData()
+        {
+            var personalDataOption = A<GDPRPersonalDataOption>();
+            var itSystemUsage = new ItSystemUsage
```

Comment:
repeated creation code.. remember to refactor and create helpers for repeated code